addons/point_of_sale/static/src/app/generic_components/inputs/input/input.xml

```diff
@@ -10,7 +10,7 @@
             </button>
             <div t-if="!props.isSmall || state.isOpen"
                 class="input-group h-100">
-                <div class="form-control input-container d-flex align-items-center py-1 bg-view"
+                <div class="form-control input-container d-flex align-items-center m-auto bg-view"
```

Comment:
remove this, but instead, make the category-container's style to have minimum height.